pkg/sql/colexec/auto_incr.go

```diff
@@ -328,6 +328,7 @@ func updateAutoIncrTable(param *AutoIncrParam, curNum uint64, name string, mp *m
 		bat.Clean(mp)
 		return err
 	}
+	param.proc.SetLastInsertID(curNum)
```

Comment:
This is a problem -- should it be added here?    given current impl, this maybe called in remote, different places.   Is the result correct?

Reply:
and what if this transaction is aborted?   You still overwrite the value?